core/frontend/src/App.vue

```diff
@@ -20,20 +20,9 @@
         <span class="d-flex flex-column align-center">
           <backend-status-checker @statusChange="changeBackendStatus" />
           <span v-if="settings.is_pirate_mode">
-            <span class="hidden-sm-and-down">Ahoy matey! You're running</span>
-            <v-icon>
-              mdi-flag-variant
-            </v-icon>
-            <v-icon>
-              mdi-skull-crossbones
-            </v-icon>
+            <v-icon>mdi-skull-crossbones</v-icon>
             Pirate Mode
```

Comment:
Maybe we should add `Running Pirate Mode` or `Pirate Mode Activated`, to make it explicit that the user has done something to have the `Pirate Mode` there.

Reply:
I'm of the opinion that the `Pirate Mode` text is enough. Anything more than that will also force the text to be significantly smaller to fit here, compromising layout/legibility.

Reply:
Valid point.

Reply:
What do you think about making the text black and bold?

![image](https://user-images.githubusercontent.com/6551040/165383454-f5e8e08e-5716-41a4-a4f1-f05774db80f6.png)

Reply:
I think we can stay with the current approach without the bold. And see how it goes with SMS.